Lesson 6 - Developer tools/code/server.js

```diff
@@ -15,6 +15,71 @@ app.get('/today', function (req, res) {
     }, 1000);
 });
 
+app.post('/numbers', function(req, res) {
+    numbers.push(+req.body);
+    
+    res.sendStatus(200);
+});
+
+app.get('/numbers', function(req, res) {
+    var response = "";
+    
+    numbers.forEach(function(elem) {
+        response += elem + ", ";
+    });
+    
+    res.send(response);
+});
+
+app.delete('/numbers', function(req, res) {
+    numbers = [];
+    
+    res.sendStatus(200);
+});
+
+app.get('/numbers/:op', function(req, res) {
+    var operation = req.params.op;
+    var response = 0;
+    
+    switch (operation) {
```

Comment:
This could be rewritten with more functional approach - without `switch` :) Try `map`.